grpc/dispatcher.go

```diff
@@ -104,116 +111,11 @@ func NewDispatcher(config DispatcherConfig) (*Dispatcher, error) {
 			errors.New("grpc dispatcher: "+responseStatus.String()))
 	}
 
-	// Get reflection response from reflection server, which contain FileDescriptorProtos
-	reflectionResponse, err := getReflectionResponse(conn, config.Service)
-	if err != nil {
-		return nil, err
-	}
-	fileDescriptorProtoBytes := reflectionResponse.GetFileDescriptorResponse().GetFileDescriptorProto()
-
-	messageDescriptor, err := getMessageDescriptor(fileDescriptorProtoBytes, config.Service, config.Method)
-	if err != nil {
-		return nil, err
-	}
-
 	dispatcher := &Dispatcher{
 		timeout:       configuredTimeout,
-		serviceMethod: fmt.Sprintf("%s/%s", config.Service, config.Method),
+		serviceMethod: fmt.Sprintf("/%s/%s", config.Service, config.Method),
```

Comment:
this is probably the most bizarre struggle i had today. It turns out that the python server cant work without the trailing `/` unlike grpc server. I thought it had to do with the codec for long time.

Check the generated proto for golang and [python](https://github.com/caraml-dev/universal-prediction-interface/blob/main/gen/python/grpc/caraml/upi/v1/upi_pb2_grpc.py#L19), it does start with a `/`

Reply:
Let me check this. It might be the issue with Python generated code itself. But strangely it works with grpcurl, ghz, etc. though

Reply:
Strange, all generated code expect prefix `/`, how come the golang code can work.

Reply:
yep i saw both golang and python has prefix `/` but somehow the golang server can function without it.